### PR TITLE
Features/another test

### PR DESCRIPTION
**Pull request by ehsanmotamedi**
Made some changes to the 2015 template's page.php and added a corresponding class.
@Lindsor Can you please code review?
@svignara Can you please code review?

### CODE REVIEW

**Comment by ehsanmotamedi** [comment]
@svignara Can you please comment on the line you're talking about?

**Comment by ehsanmotamedi** [comment]
I don't see any release notes in this pull request.

**Comment by ehsanmotamedi** [comment]
@svignara Thanks! Please verify.

File: wp-content/themes/twentyfifteen/page.php [hunk 1]
```diff
@@ -14,14 +14,16 @@
...
 	<div id="primary" class="content-area">
-		<main id="main" class="site-main" role="main">
+		<main id="main" class="site-main some-other-class" role="main">
 
 		<?php
 		// Start the loop.
 		while ( have_posts() ) : the_post();
 
 			// Include the page content template.
+			<div class="one-post some-class">
```
Comment (Lindsor) [inline wp-content/themes/twentyfifteen/page.php:24]: the php tag was never closed so no html can go in.

File: wp-content/themes/twentyfifteen/css/editor-style.css [hunk 2]
```diff
@@ -475,3 +475,7 @@ body.rtl {
 .rtl table caption {
 	text-align: right;
 }
+
+.some-class {
+	background-color: #900;
```
Comment (Lindsor) [inline wp-content/themes/twentyfifteen/css/editor-style.css:480]: we do not allow some-class to have that background color some-class is always #009

**Comment by Lindsor** [comment]
Can we restrict who sees/can click the "Merge pull request" button? because i have a thing for buttons and can see myself clicking it accidentally like 10 times.

**Comment by svignara** [comment]
@ehsanmotamedi it's true, @Lindsor has a tendency to break things

**Comment by ehsanmotamedi** [comment]
@svignara @Lindsor We can always revert this merge if we wanted to.

**Comment by ehsanmotamedi** [comment]
@svignara @Lindsor We can also protect the master branch with Github and put different rules around it.

**Comment by Lindsor** [comment]
Wow Suve the point of git is so that we all agree it can be merged. Way to go rogue...
@svignara

**Comment by svignara** [comment]
@Lindsor :expressionless: i got hacked my someone, talk about security breaches

**Pull request merged**